msgpack/index.d.ts

```diff
@@ -81,3 +81,6 @@ interface MsgPackCallbackResult {
 }
 
 declare var msgpack: MsgPackStatic;
```

Comment:
Correct thing to do here is wrap the entire thing in a namespace named `msgpack`, then keep this variable around with the following type:

```ts
declare var msgpack: msgpack.MsgPackStatic;
```

and then everything should work again.